### ISSUE TITLE
what reason for 404? all host is same

### ISSUE THREAD

**Issue by aachinakfw** [C0nw0nk/Nginx-Lua-Anti-DDoS#47]
404 Not Found
openresty

server   {
      listen 443 ssl;
	
      server_name  www.godo322.com;
      ssl on;
      ssl_certificate  /ssl/www.godo322.com.crt;
      ssl_certificate_key  /ssl/www.godo322.com.key;
      keepalive_timeout    60 ;
     access_by_lua_file anti_ddos_challenge.lua;
      location / {
      proxy_pass  https://netdisk462;
      include proxy.conf;
                 }

**Comment by Sylvvvia** [comment]
access_by_lua_file anti_ddos_challenge.lua;

You need to specify location to this file.

**Comment by C0nw0nk** [comment]
Yes it looks like you only specified the file name but not the full path so nginx can find it.

The readme is guidance you do need to specify full paths where need be.
